main/ui.js

```diff
@@ -24,8 +23,8 @@ module.exports = async function (ctx) {
   const ui = new BrowserWindow({
     title: 'Filecoin Station',
     show: false, // we show it via ready-to-show
-    width: store.get('ui.width', dimensions.width < 1440 ? dimensions.width : 1440),
-    height: store.get('ui.height', dimensions.height < 900 ? dimensions.height : 900),
```

Comment:
I removed these since nothing was writing `ui.height`, and simplified the logic

Reply:
No I totally agree, PRed [here](https://github.com/filecoin-station/filecoin-station/pull/238)